e2e/cypress/integration/messaging/message_auto_response_spec.js

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @auto_response @messaging
+
+describe('Auto Response In DMs', () => {
+    const AUTO_RESPONSE_MESSAGE = 'Out of Office';
+    const MESSAGE = 'Message';
+    let userA;
+    let userB;
+    let testTeam;
+    let offTopicChannelUrl;
+
+    before(() => {
+        // # Enable ExperimentalEnableAutomaticReplies setting
+        cy.apiUpdateConfig({TeamSettings: {ExperimentalEnableAutomaticReplies: true}});
+
+        // # Create a new team
+        cy.apiInitSetup().then(({team, user}) => {
+            userA = user;
+            testTeam = team;
+
+            // # Create a second user
+            cy.apiCreateUser().then(({user: createdUser}) => {
+                userB = createdUser;
+                cy.apiAddUserToTeam(team.id, userB.id);
+            });
+
+            offTopicChannelUrl = `/${testTeam.name}/channels/town-square`;
+        });
+    });
+
+    it('MM-T4004_1 Login as userB and set auto response message', () => {
+        // # Login as userB
+        cy.apiLogin(userB);
+
+        // # Visit off topic channel
+        cy.visit(offTopicChannelUrl);
+
+        // # Open 'Account Settings' modal and view 'Notifications'
+        cy.uiOpenAccountSettingsModal('Notifications').within(() => {
+            // # Click on 'Edit' for 'Automatic Direct Message Replies
+            cy.get('#auto-responderEdit').should('be.visible').click();
+
+            // # Click on 'Enabled' checkbox
+            cy.get('#autoResponderActive').should('be.visible').click();
+
+            // # Clear default auto response message
+            cy.get('#autoResponderMessageInput').should('be.visible').clear();
+
+            // # Enter new Auto Response Message
+            cy.get('#autoResponderMessageInput').should('be.visible').type(AUTO_RESPONSE_MESSAGE);
+
+            // # Save and close the modal
+            cy.uiSaveAndClose();
+        });
+
+        // # Logout userB
+        cy.apiLogout();
+    });
+
+    it('MM-T4004_1 Login as userA and send direct messages and verify only one auto response message is sent from userB', () => {
+        // # Login as userA
+        cy.apiLogin(userA);
```

Comment:
We're exactly matching the `it` block and the corresponding test step as written in our test management system so that we'll be able to record the rest result. With that, please combine into a single `it` block only, like
```suggestion
        // # Logout userB
        cy.apiLogout();

        // # Login as userA
        cy.apiLogin(userA);
```